docs/usage/webhook/webhook.rst

```diff
@@ -63,7 +63,7 @@ We'll be using operations from ``dffml_operations_deploy`` and ``dffml_feature_g
 
 .. code-block:: console
 
-    $ pip install dffml_operations_deploy dffml_feature_git
+    $ pip install dffml-operations-deploy dffml_feature_git
```

Comment:
When we install with pip the common convention is to use the form of the package name with `-` rather than the import format which is `_`

```suggestion
    $ pip install dffml-operations-deploy dffml-feature-git
```